packages/@aws-cdk/aws-cognito/test/integ.user-pool-explicit-props.ts

```diff
@@ -27,6 +27,10 @@ const userpool = new UserPool(stack, 'myuserpool', {
     email: true,
     phone: true,
   },
+  keepOriginal: {
```

Comment:
Updating this will require an update to the expected output file as well.

Reply:
Thanks! Updated the snapshot